raspiaudio-muse-luxe.yaml

```diff
@@ -43,6 +59,39 @@ external_components:
 
 es8388:
 
+globals:
+  - id: wifi_connected
+    type: bool
+    initial_value: "false"
+    restore_value: false
```

Comment:
I'm not sure I understand the purpose of this global?

Reply:
Is there anything I need to do about this @jesserockz ?